src/qt/receivecoinsdialog.cpp

```diff
@@ -286,6 +289,19 @@ void ReceiveCoinsDialog::copyURI()
     GUIUtil::setClipboard(uri);
 }
 
+// context menu action: copy address
```

Comment:
I understand that this comment follows the surrounding code style.
But isn't it worth to add a Doxygen comment into the header instead of such an approach?